connector_importer/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,59 @@
+This module allows to import / update records from files using the connector
```

Comment:
@sebalix this description is a bit confused/confusing. Can you split it out of the bare migration so that we can fast track it? Then you can open a PR only for docs improvement. Fine for you?

Reply:
Fine

Reply:
@simahawk PR updated, documentation will be moved to another PR once this one is merged